satellite/certdb/certdb.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package certdb
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/storj"
+)
+
+// DB stores storagenode public keys.
```

Comment:
I think it would be more accurate if this said something like "DB stores storagenode peer identities."

Reply:
updated